### ISSUE TITLE
Enumeration return object instead of array if there is only 1 enumeration

### ISSUE THREAD

**Issue by Nathan961207** [Mermade/jgeXml#12]
## Description

When parsing an xsd schema that contains an enumeration with a single value, the json schema type is an object instead of an array which makes the schema invalid.

### XSD Schema
Omitted beginning and end of XSD Schema
```xml 
    <xs:complexType name="DateType1Code">
        <xs:restriction base="xs:string">
            <xs:enumeration value="UKWN"/>
        </xs:restriction>
    </xs:complexType>
```

## Steps to Reproduce

 Parse an XSD schema containing a single enumeration
Example XSD Schema
```xml
<xs:schema elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
  <xs:element name="Address">
    <xs:complexType name="DateType1Code">
        <xs:restriction base="xs:string">
            <xs:enumeration value="UKWN"/>
        </xs:restriction>
    </xs:complexType>
  </xs:element>
</xs:schema>
```

#### Expected behavior

![image](https://user-images.githubusercontent.com/15314334/83754400-31c69c80-a66c-11ea-9ea1-1d9a48baee24.png)

#### Actual behavior

![image](https://user-images.githubusercontent.com/15314334/83754211-e14f3f00-a66b-11ea-9f2f-28752c06025f.png)

### Workaround
Add another enumeration with a value that explicitly tell the user to not use it

**Comment by MikeRalphson** [comment]
Ok, it looks like the `xs:enumeration` is not being parsed into an `enum` at all correctly. Will add a failing test case for this based on your XSD example.

**Comment by MikeRalphson** [comment]
Fixed in `jgexml@0.4.3` - thanks for reporting the bug!
